tool/build_xcframework.sh

```diff
@@ -64,6 +64,8 @@ EOF
 
   cp -Rf "${BUILD_DIR}/powersync-sqlite-core.xcframework" "powersync-sqlite-core.xcframework"
   tar -cJvf powersync-sqlite-core.xcframework.tar.xz powersync-sqlite-core.xcframework LICENSE README.md
+  # swift command used for checksum required for SPM package does not support xz so need to create a zip file
+  zip -r powersync-sqlite-core.xcframework.zip powersync-sqlite-core.xcframework LICENSE README.md
```

Comment:
We can use the zip for the Cocoapod package as well ([supports zip, tgz, bz2, txz and tar](https://guides.cocoapods.org/syntax/podspec.html#source)).

Reply:
Oh great. I've removed it and moved everything to `zip`